assets/src/components/ladderPage.tsx

```diff
@@ -117,6 +118,7 @@ const AddTabButton = ({
   return (
     <button
       className="m-ladder-page__add-tab-button"
+      data-testid="add-tab-button"
```

Comment:
One thing we've done in the past is add a `title`, which React Testing Library can also query by. The `title` property has some issues but I think as long as you don't use it to encode information that isn't available via other means, and instead to just label some symbol / button, it's fine.

Reply:
Yeah, I'd make it something that's formatted to be human-readable as it does show up as a tool tip and I believe in theory can also be referenced by things like screen readers. I usually don't involve design for things where the title seems fairly obvious but it wouldn't hurt to ask.